oracle_cdb_user.sql

```diff
@@ -0,0 +1,7 @@
+CREATE USER c##tidal_comm_user IDENTIFIED BY "replace_this_with_secure_password" account unlock;
```

Comment:
What does this commend do? It's at least highlighted weirdly?

Reply:
not a comment. broken syntax highlightling on GitHub. `c##tidal_comm_user` is the username. and `c##` is required Oracle syntax.